packages/embed/src/css/sidetab.scss

```diff
@@ -37,9 +37,11 @@
   &-button {
     position: absolute;
     top: 50%;
-    transform: rotate(-90deg);
-    transform-origin: bottom left;
-    width: 200px;
+    left: -48px;
+    transform: rotate(-90deg) translateX(-50%);
+    transform-origin: left top;
+    min-width: 100px;
+    max-width: 300px;
```

Comment:
Since the sidetab modal window has height of 580px, I think we can go up to 540px for max-width here. Its not the nicest design, but I think we should enable our customers to have longer texts there. I dont see a reason we should limit this to 300px (and for non-tech users there is a character limit in shareinguito anyway). WDYT?

<img src="https://user-images.githubusercontent.com/1561771/144019071-a339d4c2-65b9-4e81-be5c-3a0a863f260d.png" width=360 />

Reply:
To me it makes sense, I went with the 300px fixed width because I followed the 24 chars limit that is on the share page, but otherwise if not needed in this case, makes totally sense to give more width and space here 🚀